apps/glusterfs/pendingop.go

```diff
@@ -28,23 +28,31 @@ import (
 type PendingOperationType int
 
 const (
+	// These are enum and we should never change the mapping. Even if an
```

Comment:
Let's change "These are enum" to "These values are persisted". Both here and in the other location.

Reply:
done